style.css

```diff
@@ -0,0 +1,165 @@
+@media (min-width: 375px) {
+  body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 0;
+  }
+
+  * {
+    box-sizing: inherit;
+  }
+
+  .container {
+    position: relative;
+    display: flex;
+  }
+
+  .header1 {
+    display: flex;
+    position: absolute;
+    top: 44px;
+    width: 100%;
+    height: 46px;
+  }
+
+  .placeHolder {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    position: absolute;
+    height: 48px;
+    left: 12px;
+  }
+
+  .logo {
+    text-decoration: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 20px;
+    line-height: 28px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    width: 90px;
+    height: 28px;
+    color: #28352f;
+  }
+
+  .navg {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 8px;
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    right: 25px;
+    top: calc(50% - 40px / 2);
+    border-radius: 8px;
+  }
+
+  .union {
+    position: absolute;
+    width: 18px;
+    height: 12px;
+    left: 3px;
+    top: 6px;
+    background: #28352f;
+  }
+
+  .v1 {
+    position: absolute;
+    left: 16.67%;
+    right: 16.67%;
+    top: 70.83%;
+    bottom: 29.17%;
+    border: 2px solid #acb7c3;
+  }
+
+  .v2 {
+    position: absolute;
+    left: 16.67%;
+    right: 16.67%;
+    top: 50%;
+    bottom: 50%;
+    border: 2px solid #acb7c3;
+  }
+
+  .v3 {
+    position: absolute;
+    left: 16.67%;
+    right: 16.67%;
+    top: 29.17%;
+    bottom: 70.83%;
+    border: 2px solid #acb7c3;
+  }
+
+  .Headline {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 58px;
+    position: absolute;
+    width: 100%;
+    height: 726px;
+    top: 92px;
+    box-sizing: border-box;
+    background-image: url("/images/background_img.svg");
```

Comment:
- Hey, as requested by the previous reviewer, it is great adding the background image, however, your image is not rendering because of how you've specified the file path. To fix this update the path to this 
`background-image: url("./images/background_img.svg");`

Reply:
Hi @zieeco  could you please tell me on which browser you run and also dimensions on which you are checking. Because at my end its all good. Moreover, in Mobile Portfolio project mobile min-width : 375. 
<img width="465" alt="img12" src="https://user-images.githubusercontent.com/70527740/219801535-1ed97116-06d0-41f1-b587-3c9430aeef95.PNG">